pkgs/development/libraries/gtk+/gtk3-setup-hook.sh

```diff
@@ -0,0 +1,10 @@
+fixupOutputHooks+=(_gtk3CleanComments)
+
+# Clean comments that link to generator of the file
+_gtk3CleanComments() {
```

Comment:
Needed to add this because of how this file is created with `post_install.py` it  causes a circular reference.